internal/renderer/helm_test.go

```diff
@@ -0,0 +1,14 @@
+package renderer
+
+import (
+    "testing"
+
+    "gotest.tools/assert"
+)
+
+func TestToGoTemplate(t *testing.T) {
+    vars := `$VAR ${HELLO} ${WORLD:-world} ${FOOBAR?errbaz} $$`
+    result, err := toGoTemplate(vars)
+    assert.NilError(t, err)
+    assert.Equal(t, result, "{{.Values.VAR}} {{.Values.HELLO}} {{.Values.WORLD}} {{.Values.FOOBAR}} $")
```

Comment:
We're loosing the `default` here right ? (for `WORLD`)
Looking around in helm, it seems there is a way to set `default` value with `| default`, like the following `{{ .Values.WORLD | default "world" }}`

Reply:
https://github.com/moby/moby/pull/37134 may be of interest as well (and adds some possibly interesting tests)

Reply:
@vdemeester I think it's interesting to think about, do we want those default values in the template, or should these be interpolated during `init` and inserted in the `settings.yml` instead?

Reply:
@shin- good point, I'm not sure :sweat_smile: 
cc @chris-crone @silvin-lubecki @mnottale @mat007